armi/reactor/tests/test_blocks.py

```diff
@@ -1876,30 +1872,33 @@ def test_negativeVolume(self):
 class HexBlock_TestCase(unittest.TestCase):
     def setUp(self):
         _ = settings.Settings()
-        self.HexBlock = blocks.HexBlock("TestHexBlock")
+        self.hexBlock = blocks.HexBlock("TestHexBlock")
         hexDims = {"Tinput": 273.0, "Thot": 273.0, "op": 70.6, "ip": 70.0, "mult": 1.0}
         self.hexComponent = components.Hexagon("duct", "UZr", **hexDims)
-        self.HexBlock.add(self.hexComponent)
-        self.HexBlock.add(
+        self.hexBlock.add(self.hexComponent)
+        self.hexBlock.add(
             components.Circle(
                 "clad", "HT9", Tinput=273.0, Thot=273.0, od=0.1, mult=169.0
             )
         )
-        self.HexBlock.add(
+        self.hexBlock.add(
             components.Circle(
                 "wire", "HT9", Tinput=273.0, Thot=273.0, od=0.01, mult=169.0
             )
         )
-        self.HexBlock.add(
+        self.hexBlock.add(
             components.DerivedShape("coolant", "Sodium", Tinput=273.0, Thot=273.0)
         )
-        self.HexBlock.autoCreateSpatialGrids()
+        self.hexBlock.autoCreateSpatialGrids()
         r = tests.getEmptyHexReactor()
         a = makeTestAssembly(1, 1)
-        a.add(self.HexBlock)
+        a.add(self.hexBlock)
         loc1 = r.core.spatialGrid[0, 1, 0]
         r.core.add(a, loc1)
 
+    def test_cornersUp(self):
+        self.assertTrue(self.hexBlock.cornersUp())
+
```

Comment:
Can we add another test for the case where this would return `False`? Otherwise it is not clear that the new `Block.cornersUp` method is working.

Reply:
done